src/vscode-bicep/src/commands/findOrCreateActiveBicepFile.ts

```diff
@@ -162,7 +162,7 @@ async function queryCreateBicepFile(
   return uri;
 }
 
-function compareStrings(a: string, b: string): number {
+export function compareStrings(a: string, b: string): number {
```

Comment:
Please move to a separate file and rename to something like compareStringsOrdinal

Reply:
Updated here:
https://github.com/Azure/bicep/commit/e20f557e2744128b78894191d4fa28f7bcbd04a4